apps/web/components/team/TeamListItem.tsx

```diff
@@ -99,7 +99,7 @@ export default function TeamListItem(props: Props) {
             </>
           )}
           {!isInvitee && (
-            <div className="flex space-x-2 rtl:space-x-reverse">
```

Comment:
When the dropdown gets included in the same context (when not "portalled"), there is an extra space added by this util, so I removed it. Spacing does not look bad without it.